src/api/app/views/webui/package/revisions.html.haml

```diff
@@ -17,8 +17,10 @@
             - else
               %i Revision #{revision} not found
       - unless params['show_all']
-        = paginate @revisions
-        = link_to('Show all', action: 'revisions', project: @project, package: @package, show_all: 1)
+        = paginate @revisions, views_prefix: 'webui'
+        .text-center= link_to('Show all',
```

Comment:
We should add `.mt-3`, otherwise `Show all` is right under the last revision.

![preview](https://user-images.githubusercontent.com/1102934/79451055-306ee100-7fe6-11ea-879c-bc8967dba632.png)

Reply:
It's fine though when the pagination controls are present.

![preview](https://user-images.githubusercontent.com/1102934/79452522-8e9cc380-7fe8-11ea-90ee-25659ea695d2.png)

So I believe the issue is actually displaying `Show All` when it isn't needed. We should not display `Show All` if there are no pagination controls.

Reply:
You are right, @dmarcoux! Now it is not displayed if there's only one page to show.

It could even be improved adding a `Show less` button when displaying all. But I think for this PR. which is focused on the appearance, is enough.